app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -107,6 +132,39 @@ class MainActivity : BaseActivity(), SyncManager.SyncListener {
         selectedTagId = selectedId // In case selected tag wasn't found in list
     }
 
+    fun removeDrawerToggle() {
+        drawerToggle.isDrawerIndicatorEnabled = false
+        supportActionBar?.setDisplayHomeAsUpEnabled(false)
+        supportActionBar?.setHomeButtonEnabled(false)
+    }
+
+    override fun addDrawerToggle() {
+        drawerToggle.isDrawerIndicatorEnabled = true
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setHomeButtonEnabled(true)
+    }
+
+    override fun newNoteListener(uuid: String) {
+        noteFragment = NoteFragment()
+        noteFragment!!.detachListener = this
+        val bundle = Bundle()
+        bundle.putString(NoteListFragment.EXTRA_NOTE_ID, uuid)
+        noteFragment!!.arguments = bundle
+        if (findViewById(R.id.note_container) == null) {
+            removeDrawerToggle()
```

Comment:
I see why you went for the single-activity solution, but it does create weirdness around the action bar. Feel like there must be a better solution than this.